### DISCUSSION TITLE
Support for XML environmental variables parsing

### DISCUSSION THREAD

**Discussion by JLBuenoLopez** [eProsima/Fast-DDS#3377] [Ideas]
#1517 tried to introduce this feature in Fast DDS. Some work was still required so it was not merged. This ticket will keep track of this nice to have feature and the reference to the PR and the missing elements that should be included in order to merge. Specially, [this comment](https://github.com/eProsima/Fast-DDS/pull/1517#pullrequestreview-599401028) list the required changes.

**Comment by mauropasse** [comment]
https://github.com/eProsima/Fast-DDS/pull/3841
This PR should add support for this feature
